docs/nitclk/class.rst

```diff
@@ -160,7 +160,7 @@ configure_for_homogeneous_triggers
             
 
 
-        :type sessions: list of (Driver Session or nitclk.SessionReference)
+        :type sessions: (Driver Session or nitclk.SessionReference)
```

Comment:
I think this is now wrong...

Reply:
Yes, missed adding this in the nitclk metadata now that "list of " no longer added.